enabling_redfish_subscriptions_on_bare_metal.adoc

```diff
@@ -0,0 +1,93 @@
+// This is included in the following assemblies:
+//
+// TBD
+
+:_content-type: PROCEDURE
+[id='enabling_redfish_subscriptions_on_bare_metal_{context}']
+= Enabling redfish subscriptions on bare metal
+include::TBD/enabling_redfish_subscriptions_on_bare_metal.adoc[]
+:context: TBD
+
+Users of bare metal hardware may want to receive events from the baseboard management controller (BMC) to act on them in case of a hardware fault, such as an increase in temperature or removal of a device. The Redfish standard includes the ability to subscribe to events, which will cause hardware events to be sent in a particular format to a destination URI. Configuring a subscription is done using the Metal3 API.
+
+Execute the following procedure from the desired node to configure event subscriptions and alerts.
+
+[IMPORTANT]
+====
+This feature only works on physical hardware running Redfish with support for events with a vendor interface set to `redfish` or `idrac-redfish`. 
+====
+
+.Procedure
+
+. Power off the bare metal node by using the baseboard management controller, and ensure it is off.
+
+. Retrieve the user name and password of the bare metal node's baseboard management controller. 
+
+.Confirm the hardware has EventService enabled by using the iDRAC/iLO interface [Add Steps] or the following cURL command:
++
+[source,terminal]
+----
+curl https://<bmc_address>/redfish/v1/EventService --insecure -H 'Content-Type: application/json' -u "<username>:<password>"
+----
++
+. Create a configuration file for the bare metal node.
++
+[source,terminal,subs="+quotes"]
+----
+$ vim __<config_file>__.yaml
+----
++
+[source,yaml,subs="+quotes"]
+----
+apiVersion: metal3.io/v1alpha1
+kind: BMCEventSubscription
+metadata:
+name: sub-01
+namespace: openshift-machine-api
+spec:
+hostName: cnfdd4-worker-1
+destination: https://hw-event-proxy-cloud-native-events.apps.cnfdd4.t5g.lab.eng.bos.redhat.com/webhook
+context: ""
+----
++
+.Example output
+[source,terminal]
+----
+"ServiceEnabled": true,
+"Status": {
+"Health": "OK",
+"HealthRollup": "OK",
+"State": "Enabled"
+},
+----
```

Comment:
Shouldn't this be right after L30?

Reply:
Addressed in `nw-rfhe-creating_bmc_event_sub.adoc`.